### PR TITLE
Fix/wizard minor fixes accessibility and export screens

### PR DESCRIPTION
**Pull request by M-i-k-e-l**
Wizard - minor fixes and add accessibility.
Export screens.

### CODE REVIEW

File: src/components/text/index.js [hunk 1]
```diff
@@ -35,7 +35,11 @@ export default class Text extends PureBaseComponent {
     /**
      * Custom highlight style for highlight string
      */
-    highlightStyle: PropTypes.oneOfType([PropTypes.object, PropTypes.number])
+    highlightStyle: PropTypes.oneOfType([PropTypes.object, PropTypes.number]),
+    /**
+     * Turn off accessibility for this view and its nested children
+     */
+    inaccessible: PropTypes.bool
```
Comment (ethanshar) [inline src/components/text/index.js]: I thought we said we won't add it for now?

File: src/components/wizard/WizardStep.js [hunk 2]
```diff
@@ -102,7 +129,13 @@ export default class WizardStep extends PureBaseComponent {
         {index > activeIndex && <View flex style={[styles.connector, connectorStyle]}/>}
         {this.renderCircle(props)}
         {index === activeIndex && (
-          <Text text80 testID={`${testID}.label`} numberOfLines={1} style={[styles.label, {maxWidth}, labelStyle]}>
+          <Text
+            text80
+            testID={`${testID}.label`}
+            numberOfLines={1}
+            style={[styles.label, {maxWidth}, labelStyle]}
+            inaccessible
```
Comment (ethanshar) [inline src/components/wizard/WizardStep.js]: Why aren't you using accessible={false}

File: src/components/text/index.js (hunk 1 above)
Comment (M-i-k-e-l) [reply src/components/text/index.js]: Removed

File: src/components/wizard/WizardStep.js (hunk 2 above)
Comment (M-i-k-e-l) [reply src/components/wizard/WizardStep.js]: Changed

**Pull request merged**